dbms/src/Flash/Planner/plans/PhysicalJoin.cpp

```diff
@@ -248,6 +235,21 @@ void PhysicalJoin::doSchemaProject(DAGPipeline & pipeline, Context & context)
     executeExpression(pipeline, schema_project, log, "remove useless column after join");
 }
 
+void PhysicalJoin::buildPipeline(PipelineBuilder & builder)
+{
+    // Break the pipeline for join build.
+    // FIXME: Should be newly created PhysicalJoinBuild.
+    auto join_build_builder = builder.breakPipeline(shared_from_this());
+    // Join build pipeline.
+    build()->buildPipeline(join_build_builder);
+    join_build_builder.build();
```

Comment:
How to deal with the `Pipeline` returned from join_build_builder.build()?

Reply:
Afterwards, the dependencies of Probe and Build will be built by PipelineBreaker.